### ISSUE TITLE
build standalone static executable using cython and cmake?

### ISSUE THREAD

**Issue by anthrotype** [scikit-build/scikit-build#359]
I wonder if skbuild could be used (or adapted) to build a standalone executable with statically linked python interpreter, using cython --embed feature?
Or would that be out of scope?

**Comment by jcfr** [comment]
a possibility would be to improve https://github.com/python-cmake-buildsystem/python-cmake-buildsystem so that it integrates nicely with scikit-build.

**Comment by hamzamohdzubair** [comment]
@anthrotype Were you able to use skbuild or adapt skbuild to build a standalone executable with statically linked python interpreter?

**Comment by henryiii** [comment]
scikit-build (classic) after #1185 is now a thin wrapper around scikit-build-core's setuptools plugin. Closing all open issues; if this still applies, please reopen or open an issue on scikit-build-core. Thanks!
